### PR TITLE
Fix "OpenFile" hidden dialog

### PR DESCRIPTION
**Pull request by int-0**
Hi,

I'm using Debian Unstable with Qt4.8. The "OpenFile" dialog is not showed with the current code, the windows list show the dialog title, but the widget is not showed. With this little fix the dialog it's showed.

Thanks for your app!

### CODE REVIEW

File: src/mainwindow.cpp [hunk 1]
```diff
@@ -209,7 +209,7 @@ void MainWindow::ResetView()
 
 void MainWindow::OpenVideoFile()
 {
-	QString file = QFileDialog::getOpenFileName(this, tr("Open Video"));
+        QString file = QFileDialog::getOpenFileName(this, tr("Open Video"), QDir::currentPath(), tr("All files (*)"), Q_NULLPTR, QFileFialog::DontUseNativeDialog);
```
Comment (thestr4ng3r) [inline src/mainwindow.cpp]:
```suggestion
	QString file = QFileDialog::getOpenFileName(this, tr("Open Video"), QDir::currentPath(), tr("All files (*)"), Q_NULLPTR, QFileFialog::DontUseNativeDialog);
```

**Review by thestr4ng3r — CHANGES_REQUESTED** [review]

File: src/mainwindow.cpp (hunk 1 above)
Comment (thestr4ng3r) [reply src/mainwindow.cpp]: Please keep the tab

Comment (int-0) [reply src/mainwindow.cpp]: Mmm sorry... emacs chage it without asking

Comment (thestr4ng3r) [reply src/mainwindow.cpp]: use vim ;-P

**Comment by thestr4ng3r** [comment]
You don't have to close the pr, just click the "commit suggestion" button in my first comment.

**Comment by thestr4ng3r** [comment]
Thanks!

**Comment by int-0** [comment]
> You don't have to close the pr, just click the "commit suggestion" button in my first comment.

Ops sorry!

**Pull request merged**